frontend/src/app/people.service.ts

```diff
@@ -1,5 +1,8 @@
 import { Injectable } from '@angular/core';
 import { Person } from './people';
+import { query, QueryOutput } from 'rx-query';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
 
 const people: Person[] = [
```

Comment:
**chore:** delete

Reply:
This will be deleted once the fetching from the backend -functionality is done. Right now, this array is used in one of the methods.